cobbler/autoinstallgen.py

```diff
@@ -96,8 +96,8 @@ def generate_autoyast(self, profile=None, system=None, raw_data=None) -> str:
         self.api.logger.info(
             "AutoYaST XML file found. Checkpoint: profile=%s system=%s", profile, system
         )
-        runpost = '\ncurl "http://%s/cblr/svc/op/trig/mode/post/%s/%s" > /dev/null'
-        runpre = '\ncurl "http://%s/cblr/svc/op/trig/mode/pre/%s/%s" > /dev/null'
+        runpost = '\ncurl "%s://%s/cblr/svc/op/trig/mode/post/%s/%s" > /dev/null'
+        runpre = '\ncurl "%s://%s/cblr/svc/op/trig/mode/pre/%s/%s" > /dev/null'
```

Comment:
nitpick: These placeholders would benefit from having names

Reply:
Instead of naming them, I will convert this to an f-string since I see no reason to keep this up there in the method. The variable is only used once in that scope.

Reply:
Fixed via [8ec51f4](https://github.com/cobbler/cobbler/pull/3187/commits/8ec51f469ea24cc6edfff665ab6e8cc0bf8941dd)